src/Lib/Validate/Floats.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace tantrum_elastic\Lib\Validate;
+
+use tantrum_elastic\Exception\Validation;
+
+trait Floats
+{
+
+    /**
+     * Validates that the supplied value is a floating point number
+     * @param $value
+     * @param string $message
+     * @param string $exceptionClass
+     * @return bool
+     */
+    public function validateFloat($value, $message = 'Value is not a float', $exceptionClass = 'InvalidFloat')
```

Comment:
The default exceptions in the validation methods all extend the Validation abstract exception, however that might not always be appropriate, depending on the context. These methods are also likely to be used when validating the response from Elastic Search for example, whereupon we aught to throw a Transport\Server exception.